src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -141,13 +144,75 @@ const atiPageViewParams = ({
     },
   ];
 
-  const cleanedValues = pageViewBeaconValues.filter(({ value }) => value);
+  return getAtiUrl(pageViewBeaconValues);
```

Comment:
Unlike the one below, this doesnt include `process.env.SIMORGH_ATI_BASE_URL`, can they be consistent please? or rename this one to `buildATIPageTrackPath`

Reply:
Done!